modules/activemq-artemis-launch/added/drain.sh

```diff
@@ -2,7 +2,7 @@
 
 export BROKER_IP=`hostname -f`
 
-echo "drainer container ip(from hostname) is $BROKER_IP"
+echo "[drain.sh] drainer container ip(from hostname) is $BROKER_IP"
```

Comment:
one small thing, since the echo is getting modified already, maybe replace ip with hostname, I guess BROKER_IP is really BROKER_HOST.
It is just a small thing but makes it clear we are no longer dealing with ip addresses.

there are references to ip and ips in lots of the echo statements, as in it is clear we used to work with ip addresses!

Reply:
otherwise looks good, it seems sensible to rely on dns in this case. and probably means we can run with less privileges in the future.

Reply:
yep, that's confusing already. Will send a PR. Thanks!